pkgs/applications/audio/nuclear/default.nix

```diff
@@ -0,0 +1,32 @@
+{ appimageTools, lib, fetchurl }:
+let
+  pname = "nuclear";
+  version = "0.6.6";
+  name = "${pname}-v${version}";
```

Comment:
name is auto generated and should not contain the v before the version.

Reply:
@SuperSandro2000 `appimageTools.*` doesn't auto generate `name`, that would be true for `mkDerivation`.